README.md

```diff
@@ -2,72 +2,61 @@
 
 # Project Name
 
-> Description the project.
-
+> Portfolio Site Skeleton.
```

Comment:
- [ ] Kindly add a descriptive description of your project to your README file, so your project is documented professionally.

Reply:
Thank you @medaminedev66 for your honest review, please I have made the necessary changes. Let me know what you think, thank you